tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx","**/**/*.tsx"],
```

Comment:
Are we sure that the `**/**/*.tsx` line is necessary?

Reply:
removed